frontend/src/app/demo/demo-main-view/demo-main-view.component.html

```diff
@@ -1,6 +1,5 @@
 <app-okr-toolbar-bare [fixedPosition]="true">
   <span class="spacer"></span>
-  <app-language-picker></app-language-picker>
```

Comment:
Machen wir keinen Sprachwechsel auf der Demo Startseite mit dem neuen Pick Language Component?

Reply:
Nein, da muss der Dialog noch hinzugefügt werden, da es da das "Drei-Punkte Menü" nicht in mit der Taskleiste gibt. Steht jetzt aber auf dem Taskboard.